slices/tzdata.yaml

```diff
@@ -0,0 +1,300 @@
+package: tzdata
+
+slices:
+  zoneinfo:
+    essential:
+      - tzdata_africa
+      - tzdata_america
+      - tzdata_antarctica
+      - tzdata_arctic
+      - tzdata_atlantic
+      - tzdata_australia
+      - tzdata_brazil
+      - tzdata_canada
+      - tzdata_chile
+      - tzdata_etc
+      - tzdata_eurasia
+      - tzdata_indian
+      - tzdata_mexico
+      - tzdata_pacific
+      - tzdata_united-states
+      - tzdata_zoneinfo-base
+
+  zoneinfo-icu:
+    contents:
+      /usr/share/zoneinfo-icu/44/be/*:
+      /usr/share/zoneinfo-icu/44/le/*:
+
+  zoneinfo-base:
+    contents:
+      /usr/share/zoneinfo/CET:
+      /usr/share/zoneinfo/CST6CDT:
+      /usr/share/zoneinfo/EET:
+      /usr/share/zoneinfo/EST:
+      /usr/share/zoneinfo/EST5EDT:
+      /usr/share/zoneinfo/Factory:
+      /usr/share/zoneinfo/HST:
+      /usr/share/zoneinfo/iso3166.tab:
+      /usr/share/zoneinfo/leapseconds:
+      /usr/share/zoneinfo/leap-seconds.list:
+      /usr/share/zoneinfo/localtime:
+      /usr/share/zoneinfo/MET:
+      /usr/share/zoneinfo/MST:
+      /usr/share/zoneinfo/MST7MDT:
+      /usr/share/zoneinfo/posix/CET:
+      /usr/share/zoneinfo/posix/CST6CDT:
+      /usr/share/zoneinfo/posix/EET:
+      /usr/share/zoneinfo/posix/EST:
+      /usr/share/zoneinfo/posix/EST5EDT:
+      /usr/share/zoneinfo/posix/Factory:
+      /usr/share/zoneinfo/posix/HST:
+      /usr/share/zoneinfo/posix/MET:
+      /usr/share/zoneinfo/posix/MST:
+      /usr/share/zoneinfo/posix/MST7MDT:
+      /usr/share/zoneinfo/posix/PST8PDT:
+      /usr/share/zoneinfo/posix/WET:
+      /usr/share/zoneinfo/PST8PDT:
+      /usr/share/zoneinfo/right/CET:
+      /usr/share/zoneinfo/right/CST6CDT:
+      /usr/share/zoneinfo/right/EET:
+      /usr/share/zoneinfo/right/EST:
+      /usr/share/zoneinfo/right/EST5EDT:
+      /usr/share/zoneinfo/right/Factory:
+      /usr/share/zoneinfo/right/HST:
+      /usr/share/zoneinfo/right/MET:
+      /usr/share/zoneinfo/right/MST:
+      /usr/share/zoneinfo/right/MST7MDT:
+      /usr/share/zoneinfo/right/PST8PDT:
+      /usr/share/zoneinfo/right/WET:
+      /usr/share/zoneinfo/tzdata.zi:
+      /usr/share/zoneinfo/WET:
+      /usr/share/zoneinfo/zone1970.tab:
+      /usr/share/zoneinfo/zone.tab:
+
+  africa:
+    contents:
+      /usr/share/zoneinfo/Africa/*:
+      /usr/share/zoneinfo/posix/Africa/*:
+      /usr/share/zoneinfo/right/Africa/*:
+      /usr/share/zoneinfo/Egypt:
+      /usr/share/zoneinfo/Iceland:
```

Comment:
Iceland is probably in Europe.

Reply:
it is a symlink to `Africa/Abidjan`. Having it in the `eurasia` slice would create a dependency between the two slices

Reply:
I'll move it anyway. If we could handle slice conflicts, I'd just add the `Africa/Abidjan` to `eurasia`...but atm that won't work